doc/interesting-queries.md

```diff
@@ -509,11 +509,11 @@ select tx.id as tx_id, tx_out.value as tx_out_value, redeemer.units_mem, redeeme
 
 ### Get all mint scripts
 ```sql
-select redeemer.tx_id as tx_id, redeemer.units_mem, redeemer.units_steps, redeemer.fee as redeemer_fee, redeemer.purpose, ma_tx_mint.policy, ma_tx_mint.name, ma_tx_mint.quantity
+select redeemer.tx_id as tx_id, redeemer.unit_mem, redeemer.unit_steps, redeemer.fee as redeemer_fee, redeemer.purpose, ma_tx_mint.policy, ma_tx_mint.name, ma_tx_mint.quantity
```

Comment:
Hm this still doesn't run, because `policy` and `name` have moved to the `multi_asset` table. I think this should turn into something like this 
```
select redeemer.tx_id as tx_id, redeemer.unit_mem, redeemer.unit_steps, redeemer.fee as redeemer_fee, redeemer.purpose, multi_asset.policy, multi_asset.name, ma_tx_mint.quantity
  from redeemer 
    join multi_asset on redeemer.script_hash = multi_asset.policy 
    join ma_tx_mint on ma_tx_mint.ident = multi_asset.id and redeemer.tx_id = ma_tx_mint.tx_id 
      where purpose = 'mint';
```

Reply:
oh sorry for that, this second query is really only partial fix (as I fixed mainly unit(s) and step(s).

Reply:
@kderme fixed this one too